src/fontra/client/core/utils.js

```diff
@@ -345,9 +345,10 @@ export function arrayExtend(thisArray, itemsArray) {
 }
 
 export function rgbaToCSS(rgba) {
-  const channels = rgba.map((channel) => Math.round(channel * 255));
-  if (channels[3] == 255) {
-    channels.pop();
+  const channels = rgba.slice(0, 3).map((channel) => Math.round(channel * 255));
+  const alpha = rgba.length === 4 ? rgba[3] : undefined;
```

Comment:
In JS, you can just ask for out-of-bounds indices and get undefined:
```suggestion
  const alpha = rgba[3];
```

Reply:
Thank you. I changed it. Please see #1346